### DISCUSSION TITLE
깃허브 링크(아이콘) 화면 상단에도 있으면 좋겠어요

### DISCUSSION THREAD

**Discussion by da-in** [Yangjaecheon-Otter-Guardians/simple-thumbnail#133] [Ideas]
### 주제

Features

### Body

화면 컨텐츠가 많다보니 footer에 있는 깃허브 링크는 잘 안보이는 것 같아요 🥺

### BDD TASK

```bash
given 화면 상단에 깃허브 아이콘이 보인다
when 깃허브 아이콘을 클릭한다
then 깃허브 사이트로 이동한다
```

**Comment by caseBread** [comment]
좋은것같습니다!! 우선순위는 어떻게 설정해보면 좋을까요?

**Reply by da-in** [reply]
기능상 긴급한 건 아니라서, 중요도 중간 정도로 처리하면 좋을 것 같습니다.👍🏻
